jest.config.js

```diff
@@ -0,0 +1,7 @@
+const path = require("path");
+
+module.exports = {
+  watchPathIgnorePatterns: [
+    path.resolve(__dirname, "build")
```

Comment:
This is necessary because the webpack test needs to be built with webpack first. This watch ignore pattern fixes an infinite loop issue.

Reply:
Can you do this as a separate PR? Just adding the `jest.config.js` file. This seems useful on it's own.

Reply:
You don't need to ignore that path without the webpack test because other tests are not writing any files. Personally, I wouldn't add that configuration when it's not necessary. What do you think?

Reply:
Good point.